packages/grafana-schema/src/raw/team/x/team_types.gen.ts

```diff
@@ -9,16 +9,16 @@
 // Run 'make gen-cue' from repository root to regenerate.
 
 export enum Permission {
-  Admin = 4,
-  Editor = 2,
-  Viewer = 1,
+  admin = 4,
+  editor = 2,
+  viewer = 1,
```

Comment:
Wonder why this has been changed? We mostly use capitalized keys for enums in our codebase.

Reply:
They're explicitly written that way [in the kind decl](https://github.com/grafana/grafana/blob/0485cf34cda5ed6e209beabf70a80ab67f93bf42/kinds/structured/team/team_kind.cue#L30). If we want them uppercase, they need to be fixed in the input